ruff_lsp/server.py

```diff
@@ -99,6 +101,37 @@
     "-",
 ]
 
+# Arguments that are not allowed to be passed to Ruff.
+TOOL_NON_ARGS = [
+    # Arguments that enforce required behavior. These can be ignored with a warning.
+    "--force-exclude",
+    "--no-cache",
+    "--no-fix",
+    "--quiet",
+    # Arguments that contradict the required behavior. These can be ignored with a
+    # warning.
+    "--diff",
+    "--exit-non-zero-on-fix",
+    "-e",
+    "--exit-zero",
+    "--fix",
+    "--fix-only",
+    "-h",
+    "--help",
+    "--no-force-exclude",
+    "--show-files",
+    "--show-fixes",
+    "--show-settings",
+    "--show-source",
+    "--silent",
+    "--statistics",
+    "--verbose",
+    "-w",
+    "--watch",
+    # Arguments that are not supported at all, and will error when provided.
+    # "--stdin-filename",
+    # "--format",
```

Comment:
@charliermarsh why are these commented out?

Reply:
Because they aren’t flags, they’re positional arguments, so they’re followed by another argument. I figured this wasn’t worth handling since they would error anyway.